src/components/Modal/Modal.spec.tsx

```diff
@@ -97,10 +97,10 @@ describe('Modal', () => {
         userEvent.click(getByTestId('button-close'));
       });
 
-      await wait();
-
-      expect(defaultModal.onClose).toHaveBeenCalled();
-      expect(queryByTestId('card')).toBeNull();
+      await waitFor(() => {
+        expect(defaultModal.onClose).toHaveBeenCalled();
+        expect(queryByTestId('card')).toBeNull();
+      });
```

Comment:
Snuck in this fix to silence a deprecation warning from `@testing-library/react`.